src/test/ui/generator/refined-region-for-local-accesses.rs

```diff
@@ -0,0 +1,32 @@
+// edition:2018
+// run-pass
```

Comment:
Here too, it looks like `build-pass` or `check-pass` should be enough.

For #91032, I tried to use `check-pass` when I could, and then `build-pass` when I had a case where the new algorithm might remove a type that would still be in the MIR, since `build-pass` runs the check that the types MIR found were a subset of those found in typeck.

Reply:
It is updated to perform `build-pass` test.

Reply:
Thanks!